osbs/utils/__init__.py

```diff
@@ -726,3 +726,31 @@ def copy(self):
             namespace=self.namespace,
             repo=self.repo,
             tag=self.tag)
+
+
+class UserWarningsStore(object):
+    def __init__(self):
+        # asctime (platform:arch)? - name - levelname - message
+        self.__regex = r'^[\d\s,:-]+ (platform:.+ )?- \D+ - USER_WARNING - (\{.*\})$'
```

Comment:
Do we really need two underscores `__` instead of just one `_`?

Reply:
Could we just use regexp to `<whatever> - USER_WARNING - (<whatever>)$`?

USER_WARNING should be constant